lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -134,7 +135,7 @@ def playbook_on_stats(self, stats):
         # Print the timings
         for uuid, result in results:
             msg = ''
-            msg="{0:-<70}{1:->9}".format('{0} '.format(result['name']),' {0:.02f}s'.format(result['time']))
+            msg="{0:-<70}{1:->9}".format('{0} '.format(to_bytes(result['name'])),' {0:.02f}s'.format(result['time']))
             if 'path' in result:
-                msg += "\n{0:-<79}".format( '{0} '.format(result['path']))
-            self._display.display(msg)
+                msg += "\n{0:-<79}".format('{0} '.format(to_bytes(result['path'])))
+            self._display.display(to_unicode(msg))
```

Comment:
And then we can get rid of the to_unicode here because we've already changed it into unicode above.